ledger-signer/ledger_signer.go

```diff
@@ -182,6 +187,8 @@ func (s *ledgerSigner) Public() signature.PublicKey {
 // ContextSign generates a signature with the private key over the context and
 // message.
 func (s *ledgerSigner) ContextSign(context signature.Context, message []byte) ([]byte, error) {
+	fmt.Println("\nPlease, review the transaction on your Ledger device...")
```

Comment:
Ugh it seems ugly to have this in the signer implementation. Especially as this signer can be used for non-interactive cases (e.g., with the consensus signer app).

Reply:
Good point. What about guarding it with [`Isatty()`](https://github.com/oasisprotocol/oasis-core/blob/a4be9b539d2d9c4dffaddf70b15eee16fac8920f/go/oasis-node/cmd/common/isatty.go#L8-L9)?

Reply:
Wouldn't it be better if the CLI use case included a generic message like "You may need to review the transaction on your device if you use a hardware-based signer..."?

Reply:
Yes, that would work.

Reply:
Moved the change to: https://github.com/oasisprotocol/oasis-core/pull/3149